src/process/processinterfaces.ts

```diff
@@ -213,7 +213,7 @@ export interface ITaskExtensions {
   anonymousStart?: boolean;
   anonymousStartUserId?: string;
 
-  messageEventType?: string;
+  messageEventType?: string; // "mail" | "webhook" - defaults to mail if not explicitely set
```

Comment:
Shouldn't this be something like this?
```suggestion
  //*
   * Defaults to mail if not explicitely set
   */
  messageEventType?: "mail" | "webhook"
```

Reply:
The problem is that BpmnProcess.addOrUpdateExtension accepts only strings - so everybody can call the method and set any string. Also when parsing the xml, we had to cast from a string to the expected type.
I think we should make the addOrUpdateExtension generic to allow for type checks, but for now we cannot guarantee that it is only called with the correct values.